plugins/providers/docker/action/create.rb

```diff
@@ -156,6 +155,16 @@ def forwarded_ports(include_ssh=false)
 
           result
         end
+
+        def container_name
```

Comment:
Provide a bit more information in the method name, for example: `generate_container_name`. This will have the added benefit in this changeset of not masking the method name above where it is used.